### PR TITLE
upgrade to ODP API v1.45.1.0

### PR DESCRIPTION
**Pull request by JereLeppanen**

### CODE REVIEW

File: scripts/check-dpdk.sh [hunk 1]
```diff
@@ -9,10 +9,10 @@ fi
 cd $(readlink -e $(dirname $0))/..
 
 # Build ODP
-git clone https://github.com/OpenDataPlane/odp-dpdk --branch v1.41.0.0_DPDK_22.11 --depth 1
+git clone https://github.com/OpenDataPlane/odp-dpdk --branch v1.45.1.0_DPDK_22.11 --depth 1
 pushd odp-dpdk
 ./bootstrap
-./configure --prefix=$(pwd)/install --without-tests --without-examples
+./configure --prefix=$(pwd)/install --enable-dpdk-shared --without-tests --without-examples
```
Comment (JannePeltonen) [inline scripts/check-dpdk.sh:15]: Why this change? Othewise this PR looks good to me.

Comment (JereLeppanen) [reply scripts/check-dpdk.sh:15]: Static linking of dpdk became the default in odp-dpdk already a couple of years ago. This change is just a small convenience so that the script works with distro dpdk, which is probably shared only, so that there's no need to compile dpdk. If one does want to compile dpdk, then there's the devbuild script for that.

**Review by JannePeltonen — APPROVED** [review]

**Comment by bogdanPricope** [comment]
Are you guys still doing this? Why?

Best regards,
Bogdan
